lang/cs/Org.Apache.REEF.Client/YARN/YARNREEFClient.cs

```diff
@@ -98,13 +99,52 @@ public IJobSubmissionResult SubmitAndGetJobStatus(JobRequest jobRequest)
         {
             var application = await _yarnClient.GetApplicationAsync(appId);
 
-            var msg = string.Format("application status {0}, Progress: {1}, trackingUri: {2}, Name: {3}, ApplicationId: {4}, State {5}.",
-                application.FinalStatus, application.Progress, application.TrackingUI, application.Name, application.Id, application.State);
+            var msg =
```

Comment:
Please move the string interpolation into the `log` call. That way, it doesn't have to be evaluated if the log line isn't actually logged.